arrays.dd

```diff
@@ -659,10 +655,27 @@ int[3] foo;
 int x = foo[3]; // error, out of bounds
 ---------
 
-        $(P Insertion of array bounds checking code at runtime should be
-        turned on and off
-        with a compile time switch.
-        )
+    $(P An out of bounds memory access will cause undefined behavior,
+        therefore array bounds check is normally enabled in $(D @safe) function.
+        The runtime behavior is the part of language semantics.
+    )
+
+    $(P See also $(DDSUBLINK function, safe-functions, Safe Functions).)
+
+$(H3 $(LNAME2 disable-bounds-check, Disabling Array Bounds Checking))
+
+    $(P Insertion of array bounds checking code at runtime may be
+        turned off with compiler switch.
```

Comment:
> Also do mention the switch here.

That's intentional. This is the spec text, so I think it should not display dmd-specific switch name "-boundscheck".